app/views/providers/legal_aid_applications/search.html.erb

```diff
@@ -4,16 +4,13 @@
 
     <%= render partial: "shared/error" if @error %>
 
-    <%= form.govuk_fieldset legend: { text: page_title, tag: "h1", size: "xl" } do %>
-      <div class="govuk-!-padding-bottom-4"></div>
-      <%= form.govuk_text_field(
-            :search_term,
-            label: { text: :search_term, hidden: true },
-            caption: { text: t(".hint") },
-            width: "two-thirds",
-            value: @search_term,
-          ) %>
-    <% end %>
+    <%= form.govuk_text_field(
+          :search_term,
+          label: { text: "Search applications", size: "xl", tag: "h1" },
+          hint: { text: t(".hint") },
```

Comment:
My mistake, should this value be `caption` or `hint`?

Reply:
No, it should be a hint. This is what it looked like before:

<img width="1058" alt="image" src="https://github.com/ministryofjustice/laa-apply-for-legal-aid/assets/13471320/73de142f-4824-4429-b22c-11a14ceca7b3">

And now:

<img width="1019" alt="image" src="https://github.com/ministryofjustice/laa-apply-for-legal-aid/assets/13471320/21756ee2-d225-492e-85cc-75066e9ac646">

So I think it's always been a hint. The caption (which would be above some text) was being used as a hint

Reply:
All good then